README.md

```diff
@@ -1,48 +1,27 @@
-![Terraform Provider Iterative](https://static.iterative.ai/img/cml/banner-terraform.png)
+![TPI](https://static.iterative.ai/img/cml/banner-terraform.png)
 
-# Iterative Provider [![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
+# Terraform Provider Iterative
 
-The Iterative Provider is a Terraform plugin that enables full lifecycle
-management of computing resources for machine learning pipelines, including GPUs, from your favorite cloud vendors.
+[![](https://img.shields.io/badge/-documentation-5c4ee5?logo=terraform)](https://registry.terraform.io/providers/iterative/iterative/latest/docs)
 
-The Iterative Provider makes it easy to:
+- **Orchestrate Resources**: create cloud compute & storage resources without reading pages of documentation
+- **Sync & Execute**: move data & run code in the cloud with minimal configuration
+- **Low cost**: auto-recovery from spot/preemptible instances to vastly reduce cost
+- **No waste**: auto-cleanup unused resources
+- **No lock-in**: switch between cloud vendors with minimal configuration
 
-- Rapidly move local machine learning experiments to a cloud infrastructure
-- Take advantage of training models on spot instances without losing any progress
-- Unify configuration of various cloud compute providers
-- Automatically destroy unused cloud resources (compute instances are terminated on job completion/failure, and storage is removed when results are downloaded)
+Iterative's Provider is a [Terraform](https://terraform.io) plugin built with machine learning pipelines in mind. It enables full lifecycle management of computing resources (including GPUs) from several cloud vendors:
 
-The Iterative Provider can provision resources with the following cloud providers and orchestrators:
-
-- Amazon Web Services
+- Amazon Web Services (AWS)
 - Microsoft Azure
-- Google Cloud Platform
-- Kubernetes
-
-## Documentation
-
-See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide to learn how to use the Iterative Provider. More details on configuring and using the Iterative Provider are in the [documentation](https://registry.terraform.io/providers/iterative/iterative/latest/docs).
+- Google Cloud Platform (GCP)
+- Kubernetes (K8s)
 
-## Support
+The aim is to easily (minimal configuration unified across cloud vendors) move local experiments to the cloud, use low-cost spot instances without losing progress, and avoid being charged for unused cloud resources (terminate compute instances upon job completion/failure, and remove storage upon download of results).
```

Comment:
> _without losing progress_

Seems a bit unspecific. Would “resuming” be a more familiar concept than “not losing progress”?

Reply:
> _easily (minimal configuration unified across cloud vendors) move_

That parenthesis broke my focus in a thousand pieces 🤯

Reply:
> broke my focus

ostentatious lengthy gorgeously prevaricating adjective chaining FTW

```suggestion
With a minimal configuration unified across cloud vendors, the aim is to easily move local experiments to the cloud, transparently resume from interrupted low-cost spot instances, and avoid being charged for unused cloud resources (terminate compute instances upon job completion/failure, and remove storage upon download of results).
```

Reply:
I like it